ddtrace/contrib/integration_registry/README.md

```diff
@@ -0,0 +1,72 @@
+# Datadog Python APM Integration Registry
+
+This directory contains the canonical registry of integrations supported by `dd-trace-py`.
+
+## Purpose
+
+The `registry.yaml` file serves as a centralized, machine-readable source of truth for metadata about each integration within the `ddtrace/contrib/internal/` directory. This metadata includes essential information about dependencies and version compatibility, intended primarily for internal tooling and potentially for documentation generation.
+
+## Format
+
+The registry is stored as a single YAML file: `registry.yaml`.
+
+It consists of a root object with a single key, `integrations`, which holds a list of integration definition objects. Each integration object represents one directory found under `ddtrace/contrib/internal/`.
+
+## Schema and Fields
+
+Each integration entry in the `integrations` list adheres to the following structure (defined formally in `_registry_schema.json`):
+
+**Required Fields:**
+
+*   **`integration_name`** (String):
+    *   The canonical, lowercase, snake\_case name of the integration.
+    *   This MUST match the corresponding directory name within `ddtrace/contrib/internal/`.
+    *   Example: `flask`, `redis`, `asyncio`
+*   **`is_external_package`** (Boolean):
+    *   `true`: If the integration instruments a third-party library typically installed via pip (e.g., `flask`, `requests`, `psycopg`).
+    *   `false`: If the integration instruments a Python standard library module (`asyncio`, `logging`), or an internal ddtrace helper (`ddtrace_api`).
```

Comment:
ddtrace_api is available on PyPI so I don't think it qualifies as an "internal helper"